torch/_torch_docs.py

```diff
@@ -10549,6 +10551,13 @@ def merge_dicts(*dicts):
         for CPU tensor types and the current CUDA device for CUDA tensor types.
     {requires_grad}
     {check_invariants}
+    is_coalesced (bool, optional): When specified, set the coalesced
+        flag to :attr:`is_coalesced`. If ``True``, the caller is
+        responsible for providing tensor indices that correspond to a
+        coalesced tensor that is a prerequisite for various algoritms
+        that process tensor indices to produce expected results.
+        Default: if None, the coalesced flag may be estimated using
+        the conservative value ``False`` as an initial state.
```

Comment:
We should explicitly note the danger zone here, something like: 

```
If the `check_invariants` flag is false no error will be raised if the prerequisites are not met and this will likely lead to silently incorrect results. 
```

Should cover it

Reply:
Applied, thanks!